tests/unit/vdom.ts

```diff
@@ -71,6 +73,10 @@ describe('vdom', () => {
 	afterEach(() => {
 		consoleStub.restore();
 		resolvers.restore();
+		for (let spy of spys) {
```

Comment:
could use sinon.sandbox instead for this (then we wouldn't need the for and it's useful for other things), but not blocking this PR going in.

Reply:
That would be a good refactor for this test suite 👍